composer.json

```diff
@@ -16,7 +16,8 @@
     },
     "suggest": {
         "dflydev/markdown": "~1.0",
-        "erusev/parsedown": "~1.0"
+        "erusev/parsedown": "~1.0",
+        "league/commonmark": "*"
```

Comment:
Also: please remember this is the 'suggest' section; the value after the package name is a free form text field and not a version number